bin/mt-load-check

```diff
@@ -12,12 +12,12 @@ esac
 case $servorws in
 	ws*)
 		echo Workstation - activating GUI env
-		conda activate lsdc-gui-2022-1.1
+		PYTHONTOUSE=/opt/conda_envs/lsdc-gui-2022-1.1/bin/python
```

Comment:
An environment variable name like PYTHONENV or PYTHON_ENV may be more readable

Reply:
I agree - looking back at this, at first glance, I saw it as PYTHONHOUSE!

Reply:
would you be happy with PYTHON_BINARY or CONDA_ENV_PYTHON_BINARY? or is there something else that you think would be better?

Reply:
ok, going with `CONDA_ENV_BASE`, and bin/python will be appended to the directory name to define the python binary location